.gitignore

```diff
@@ -20,4 +20,5 @@ vendor/
 *.env
 
 # Ignore VERSION.txt
-VERSION.txt
\ No newline at end of file
+VERSION.txt
+history.mp4
```

Comment:
This will most likely cause a conflict, since we already added it in `development` but we'll see.